services/cd-service/pkg/repository/transformer.go

```diff
@@ -135,7 +135,16 @@ func GaugeEnvAppLockMetric(fs billy.Filesystem, env, app string) {
 	}
 }
 
-func UpdateDatadogMetrics(state *State, changes *TransformerResult) error {
+func GaugeDeploymentMetric(_ context.Context, env, app string, timeInMinutes float64) error {
+	if ddMetrics != nil {
+		// store the time since the last deployment in minutes:
+		err := ddMetrics.Gauge("lastDeployed", timeInMinutes, []string{"app:" + app, "env:" + env}, 1)
```

Comment:
Can you please use the same tag names as in https://github.com/freiheit-com/kuberpult/blob/main/services/rollout-service/pkg/metrics/metrics.go#L152